src/views/admin/CreateExamRegForm.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+    <div v-if="busy" class="flex h-screen">
+        <div class="m-auto">
+            <loading-component />
+        </div>
+    </div>
+    <div v-else class="w-full lg:mt-20 mt-8 bg-gray-300 mx-auto h-screen">
+        <button class="flex items-center mb-4 navigation-link" @click="back">
+            <i class="fas text-xl fa-chevron-left"></i>
+            <span class="font-bold text-sm ml-1">Back</span>
+        </button>
+        <h1 class="text-2xl font-medium text-gray-700 mb-8">{{ heading }}</h1>
+        <div>
+            <ExRegInfoSection v-model:name="examRegName" v-model:valid="nameValid" :existing-names="existingExamRegNames" />
+            <ExRegModuleSection v-model:modules="selectedModules" :existing-modules="existingModules" />
+            <section class="border-t-2 py-8 border-gray-400 lg:mt-8">
+                <div class="hidden sm:flex justify-between">
+                    <div class="flex justify-end items-center">
+                        <button id="cancel" type="button" class="w-32 mr-6 btn btn-blue-secondary" @click="back">Cancel</button>
+                        <button
+                            id="createExamReg"
+                            :disabled="!canCreate"
+                            style="width: 18rem"
+                            class="btn btn-blue-primary"
+                            @click="createExamReg"
+                        >
+                            Create Examination Regulation
+                        </button>
+                    </div>
+                </div>
+
+                <!-- different button layout for mobile -->
+                <div class="sm:hidden">
+                    <button id="mobileCancel" type="button" class="mb-4 w-full btn btn-blue-secondary" @click="back">Cancel</button>
+                    <button id="mobileCreateCourse" :disabled="!canCreate" class="mb-4 w-full btn btn-blue-primary" @click="createExamReg">
+                        Create Examination Regulation
```

Comment:
```suggestion
                        Create Exam Regulation
```
In the navbar you use the short form "exam". We should use this consistently.